packages/tools/pie-icons/bin/build.js

```diff
@@ -1,14 +1,14 @@
 const { execSync } = require('child_process');
 
 console.log('Clearing old svgs');
-execSync('npx rimraf src/assets/optimised');
+execSync('npx rimraf src/assets/_optimised');
 
-console.log('Making src/assets/optimised directory');
-execSync('mkdir "src/assets/optimised"');
+console.log('Making src/assets/_optimised directory');
+execSync('mkdir "src/assets/_optimised"');
 
 // Process SVG files (Optimise)
 console.log('Optimising SVGS');
-execSync('npx babel-node bin/process-svgs.js');
+execSync('npx babel-node bin/process-svgs.js', { stdio: 'inherit' });
```

Comment:
out of interest what does this arg do?

Reply:
It passes any log output up to the parent process, so you can see it in the shell output (otherwise it gets suppressed)